registry/storage/linkedblobstore.go

```diff
@@ -46,7 +46,7 @@ type linkedBlobStore struct {
 var _ distribution.BlobStore = &linkedBlobStore{}
 
 func (lbs *linkedBlobStore) Stat(ctx context.Context, dgst digest.Digest) (distribution.Descriptor, error) {
-	return lbs.blobAccessController.Stat(ctx, dgst)
+	return lbs.blobStore.statter.Stat(ctx, dgst)
```

Comment:
Related thread [here](https://digitalocean.slack.com/archives/CMXK6M39D/p1678127940419069) where I was asking about the unknown/missing blob.

If I understood it correctly, [Wayne’s PR](https://github.com/digitalocean/docker-distribution/pull/5) made it not possible to pull some images that could be missing layer links and caused a problem. So you reverted the change [here](https://github.com/distribution/distribution/commit/37ab304645167e4103c58e05100ece6a42796cd0). It looks like our docker-distribution still keeps the `lbs.blobStore.statter.Stat()` which references the actual blob so we should not have problem fetching a blob without the layer links.

My thoughts now are that:
1. Given how we get the 404 `blob unknown to registry` despite using `lbs.blobStore.statter.Stat()`, I am assuming  [DOCR-590](https://jira.internal.digitalocean.com/browse/DOCR-590) is a different problem.
2. As to whether it is now safe to use `lbs.blobAccessController.Stat()`, I need to look further into upstream code to see any relevant fixes.